src/syscheckd/src/db/tests/db/ComponentTest/dbInterface/dbTest.h

```diff
@@ -53,7 +84,7 @@ class DBTestFixture : public testing::Test {
                         300,
                         mockSyncMessage,
                         mockLoggingFunction,
-                        MAX_FILE_LIMIT,
+                        5000,
```

Comment:
Could you add a constant for this limits? this is more clear to see!

Reply:
Done